src/com/pentalog/pentastagiu/homework2/ex3/Main.java

```diff
@@ -0,0 +1,27 @@
+package ex3;
+
+class Main {
+
+    public static void main(String[] args) {
+
+        User user1 = new User("David", "david3@gmail.com");
+
+        User user2 = new User("Diana", "dianaaa@gmail.com");
+
+        Message message1 = new Message("Description", "User3");
```

Comment:
cele 2 clase: User si Message sun decuplate, tinand cont ca un mesaj este asociat unui User, m-as astepta ca aceasta relatie sa fie modelata ori prin existenta in User a unei liste de Message (un user are asociate mesajele scrise de el in Board) ori prin declararea in Message a relatiei: in Message sa ai un author care reprezinta de fapt un user(private User author) si sa te folosesti de aceasta relatie in board cand printezi mesajele